### PR TITLE
chore: update base::Bind in app module

### PR DESCRIPTION
**Pull request by codebytere**
#### Description of Change

Update `base::Bind` to `base::BindRepeating` and `base::BindOnce` where applicable.

cc @deepak1556 @MarshallOfSound 

#### Checklist

- [x] PR description included and stakeholders cc'd
- [x] `npm test` passes
- [x] PR title follows semantic [commit guidelines](https://github.com/electron/electron/blob/master/docs/development/pull-requests.md#commit-message-guidelines)
- [x] [PR release notes](https://github.com/electron/clerk/blob/master/README.md) describe the change in a way relevant to app developers, and are [capitalized, punctuated, and past tense](https://github.com/electron/clerk/blob/master/README.md#examples).

#### Release Notes

Notes: none

### CODE REVIEW

**Comment by codebytere** [comment]
Depends on https://github.com/electron/electron/pull/17941

File: atom/browser/api/atom_api_app.cc [hunk 1]
```diff
@@ -1063,8 +1065,9 @@ void App::ImportCertificate(const base::DictionaryValue& options,
         base::Value::ToUniquePtrValue(options.Clone()));
     CertificateManagerModel::Create(
         browser_context.get(),
-        base::Bind(&App::OnCertificateManagerModelCreated,
-                   base::Unretained(this), base::Passed(&copy), callback));
+        base::BindRepeating(&App::OnCertificateManagerModelCreated,
+                            base::Unretained(this), base::Passed(&copy),
```
Comment (deepak1556) [inline atom/browser/api/atom_api_app.cc]: can you try `std::move` instead of `base::Passed` here.

File: atom/browser/api/event_emitter.h [hunk 2]
```diff
@@ -51,25 +51,25 @@ class EventEmitter : public Wrappable<T> {
   template <typename... Args>
   bool EmitCustomEvent(const base::StringPiece& name,
                        v8::Local<v8::Object> event,
-                       const Args&... args) {
+                       Args&&... args) {
```
Comment (deepak1556) [inline atom/browser/api/event_emitter.h]: Does this mean that emit arguments should always be movable ?

File: atom/browser/api/atom_api_app.cc (hunk 1 above)
Comment (deepak1556) [reply atom/browser/api/atom_api_app.cc]: Looks like the change needs to be reverted, cannot forward in repeating variation.

File: atom/browser/api/event_emitter.h (hunk 2 above)
Comment (zcbenz) [reply atom/browser/api/event_emitter.h]: For template args, this means _universal reference_ (or _forward reference_):
https://isocpp.org/blog/2012/11/universal-references-in-c11-scott-meyers

**Pull request merged**